### PR TITLE
Added dark light mode first

### PR DESCRIPTION
**Pull request by mattBman23**
Decided not to change the homepage too much, else the real owners would have wasted their time on it. Added dark and light mode.

### CODE REVIEW

File: client/components/homePage/homePage.jsx [hunk 1]
```diff
@@ -6,24 +6,15 @@ import MonthlySubscr from './MonthlySubscr.jsx';
 import Container from 'react-bootstrap/Container';
```
Comment (ShinanSun) [inline client/components/homePage/homePage.jsx:7]: we don't need monthly subscription anymore on the home page? and the about section?

**Review by ShinanSun — COMMENTED** [review]
good job! can't wait to see what it look like!

File: client/components/homePage/homePage.jsx (hunk 1 above)
Comment (mattBman23) [reply client/components/homePage/homePage.jsx:7]: Making the about page now

**Review by ShinanSun — APPROVED** [review]

**Review by daniel-he-dev — APPROVED** [review]
lgtm

**Comment by jfleming9357** [comment]
I appreciate the effort put into dark/light mode however there wasn't even an attempt at maintaining login/logout functionality. Its literally just a button that says 'Logout' that doesn't even have an onClick function. And if you are logged out there is quite literally no way to log back in. Its still just the logout button that doesn't do anything.

**Pull request merged**